packages/pegasus/globals.d.ts

```diff
@@ -0,0 +1,2 @@
+declare var makeKind: function;
```

Comment:
This probably belongs in swingset. I ran across the lack of it recently too.

Reply:
I just copied from `packages/treasury`.